### PR TITLE
feat(core): match any marker 🙀

### PR DESCRIPTION
**Pull request by srl295**
- add a regex form that matches any marker

For: #9119

@keymanapp-test-bot skip

### CODE REVIEW

File: common/web/types/src/ldml-keyboard/pattern-parser.ts [hunk 1]
```diff
@@ -64,6 +64,9 @@ export class MarkerParser {
   public static readonly MAX_MARKER_INDEX = constants.marker_max_index;
   /** Max count of markers */
   public static readonly MAX_MARKER_COUNT = constants.marker_max_count;
+  /** Expression that matches any marker */
+  public static readonly ANY_MARKER_MATCH =
+    this.SENTINEL + this.MARKER_CODE + `[\\u0001-\\u${this.MAX_MARKER_INDEX.toString(16)}]`;
```
Comment (srl295) [inline common/web/types/src/ldml-keyboard/pattern-parser.ts:69]: At least the end of the range is dynamic. Or is it worth importing left-pad here for the beginning?

Comment (mcdurdin) [reply common/web/types/src/ldml-keyboard/pattern-parser.ts:69]: I'm not sure I understand the comments here?

**Review by mcdurdin — APPROVED** [review]

File: common/web/types/src/ldml-keyboard/pattern-parser.ts (hunk 1 above)
Comment (srl295) [reply common/web/types/src/ldml-keyboard/pattern-parser.ts]: @mcdurdin sorry, it was obscure.

I was joking about whether I should `require('left-pad')`  (of infamy https://www.theregister.com/2016/03/23/npm_left_pad_chaos/ ) in order to convert `1` into `0001`.

Comment (mcdurdin) [reply common/web/types/src/ldml-keyboard/pattern-parser.ts]: Surely we should define `MIN_MARKER_INDEX` as 1. Then you might get your chance to left pad. But with `padStart` please 🤣

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.187-alpha](https://keyman.com/downloads/releases/alpha/17.0.187)

**Pull request merged**